mmv1/templates/terraform/post_delete/private_cloud.go.erb

```diff
@@ -0,0 +1,39 @@
+privateCloudPollRead := func(d *schema.ResourceData, meta interface{}) transport_tpg.PollReadFunc {
```

Comment:
Is this code definitely necessary? The generated code should already be waiting for the delete operation to complete. It doesn't seem to cause any issues so it's not a blocker, but it could be nice to remove in a fast-follow if possible.

Reply:
its is required because operation completes immediately and PC is deleted in background.

Reply:
Quoting from docs
```
A PrivateCloud resource scheduled for deletion has PrivateCloud.state set to DELETED and expireTime set to the time when deletion is final and can no longer be reversed. The delete operation is marked as done as soon as the PrivateCloud is successfully scheduled for deletion (this also applies when delayHours is set to zero), and the operation is not kept in pending state until PrivateCloud is purged.
```

Reply:
💯 thanks for the explanation!